modules/core/src/main/java/org/apache/ignite/configuration/ClientConnectorConfiguration.java

```diff
@@ -349,6 +357,31 @@ public ClientConnectorConfiguration setSelectorCount(int selectorCnt) {
         return this;
     }
 
+    /**
+     * Gets maximum allowed number of active connections per node.
+     * <p>
+     * Defaults to {@link #DFLT_MAX_CONNECTION_CNT}.
+     * Zero means no limit.
+     * Negative values are not allowed.
+     *
+     * @return Maximum allowed number of active connections.
+     */
+    public int getMaxConnectionCnt() {
```

Comment:
Public API properties should not be abbreviated - see other properties here (`setSelectorCount`) and in other classes (`IgniteConfiguration#setNetworkSendRetryCount`).

Reply:
Fixed.